tests/osgi-tests/src/test/java/org/wso2/extension/siddhi/io/http/test/osgi/BasicAuthTrue.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fix licence year

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139